### PR TITLE
Add mobile homepage

### PR DESCRIPTION
**Pull request by OsvaldoBC**
Hello ! in this pull request I am adding  :

- The setup portfolio project
- Updated the Readme file
- Add the index.html and css files following the Figma template 1
- Checked and fixed lintern errors

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  color: #6070ff;
+  font-size: 18px;
+}
+
+.logo-osvaldo {
+  text-decoration: none;
+}
```
Comment (franclobo) [inline css/styles.css]: - [ ] Please, 🙏🏽 add the correct weight to the font of your logo, to look similar to the template:
 `font-weight: 700;`

> You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          <h1 class="title">I'm Osvaldo Barrios! Glad to see you!</h1>
+          <p class="description"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </section>
+        <section>
+          <h5 class="connect">LET'S CONNECT</h5>
+          <ul class="social-media">
+              <li><a href="https://www.linkedin.com/in/"><img src="./img/Linkedin.png" alt="Linkedin"></a></li>
+              <li><a href="https://www.github.com/OsvaldoBC"><img src="./img/Vector1.png" alt="twitter"></a></li>
+              <li><a href="https://www.twitter.com/@osvaldoBarrio20"><img src="./img/Vector3.png" alt="twitter"></a></li>
+          </ul>
```
Comment (franclobo) [inline index.html]: - [ ] Kindly, complete the icons that are used in the Figma template 🤝 : 
image

- Yours:

image

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+.navbar {
+  text-decoration: none;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  color: #6070ff;
+  font-size: 18px;
+}
```
Comment (franclobo) [inline css/styles.css]: - [ ] Kindly, implement the correct margins to the navbar for a better user experience. 👍🏽 

image

- Figma Template

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @OsvaldoBC, 

Good job so far!
Keep it on you are doing it well 👏🏽 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors. ✔
- Descriptive README ✔
- Descriptive PR ✔
- Correct Github flow ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- [ ] Almost all your comments are descriptive, but `First comment`, `Second comment`, etc, are not descriptive.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OsvaldoBC** [comment]
hello @franclobo  I already fixed the items solicited. Thanks for your help!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        <section>
+          <h1 class="title">I'm Osvaldo Barrios! Glad to see you!</h1>
+          <p class="description"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </section>
+        <section>
+          <h5 class="connect">LET'S CONNECT</h5>
+          <ul class="social-media">
+              <li><a href="https://www.twitter.com/@osvaldoBarrio20"><img src="./img/Vector3.png" alt="twitter"></a></li>
+              <li><a href="https://www.linkedin.com/in/"><img src="./img/Linkedin.png" alt="Linkedin"></a></li>
+              <li><a href="https://www.microverse.org/"><img src="./img/vector2.png" alt="m"></a></li>
```
Comment (Kamba56) [inline index.html:32]: - [Optional] This image doesn't display properly, kindly look into it, please.

File: .github/workflows/linters.yml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    name: Webhint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@7.x
```
Comment (Kamba56) [inline .github/workflows/linters.yml:31]: - [Optional] Kindly consider installing linters properly. You can do that by copying the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) repo to your own repository and run `npm install --save-dev hint@7.x` in your terminal to install webhint, then run `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x` in your terminal to install stylint.

**Review by Kamba56 — APPROVED** [review]
Hi @OsvaldoBC, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
ChopApproveCopApproveGIF

## Highlights
- [x] Meaningful commit messages
- [x] Correct use of GitHub flow
- [x] Meaningful and descriptive PR title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**